app/src/main/java/com/xmartlabs/scasas/geoquiz/module/Question.java

```diff
@@ -0,0 +1,46 @@
+package com.xmartlabs.scasas.geoquiz.module;
+
+/**
+ * Created by scasas on 1/25/17.
+ */
+
+public class Question {
+  private int textResId;
+  private boolean answerTrue;
+
+  public int getTextResId() {
+    return textResId;
+  }
+
+  public boolean isAnswerTrue() {
+    return answerTrue;
+  }
+
+  public static final class Builder {
+    private int textResId;
+    private boolean answerTrue;
+
+    public Builder() {
```

Comment:
Just to provide more context, Java creates by default a public non-args constructor to all classes that don't declare a constructor.